### PR TITLE
Remove unused "Less than 40 hours" and "volunteer team" application flags

### PR DESCRIPTION
**Pull request by ramonh**
Since we're switching to having part-and full time teams, this flag is no longer needed.

### CODE REVIEW

**Comment by nerdyvaishali** [comment]
Thanks @ramonh for working on this. Looks good to me. 

I think removing volunteer flags was also in that list I made along with this stuff. Although I was under the impression that you did that change at lot of other parts of the reviewing system. All I can see at the moment is that the application table have column 'Volunteer (h per week)' which I'm not sure why is named as 'volunteer'. 🤔

On a side note, thanks @klappradla on accidentally clearing one of my confusion. I was also wondering why same constant is defined at two places. 🙂

**Comment by ramonh** [comment]
Thanks for the feedback, all! I'm on it 👍

**Comment by ramonh** [comment]
And done! Could you please take another look? ✌️

File: app/models/application.rb [hunk 1]
```diff
@@ -5,14 +5,7 @@ class Application < ApplicationRecord
...
   MENTOR_PICK_WEIGHT = ENV['MENTOR_PICK_WEIGHT'] || 2
-  FLAGS = [:remote_team,
-          :volunteering_team,
-          :selected,
-          :male_gender,
-          :zero_community,
-          :age_below_18,
-          :less_than_two_coaches,
-          :less_than_40_hours_a_week]
+  FLAGS = Selection::Table::FLAGS + [:selected]
```
Comment (klappradla) [inline app/models/application.rb]: Good job 👍 

To be honest, I fear that skipping the `selected` flag on the table was probably a mistake - I think it's fine to just use the exact same collection in both cases 🤞

File: app/views/reviewers/applications/index.html.slim [hunk 2]
```diff
@@ -21,7 +21,7 @@ table#applications.table.table-striped.table-bordered.table-condensed.table-hove
       th Location
       th = link_to_ordered 'Coaching Company', :coaching_company
       th.text-center colspan="2" Money ($)
-      th.text-center colspan="2" Volunteer (h per week)
+      th.text-center colspan="2" Hours per week
```
Comment (klappradla) [inline app/views/reviewers/applications/index.html.slim]: ✔️

File: app/models/application.rb (hunk 1 above)
Comment (ramonh) [reply app/models/application.rb]: I'm not sure. I'm all for merging the flags, and adding the `:selected` one to the selection table won't break anything, but I wonder if it makes sense, as selected teams would never show up during the selection process, wouldn't they?

Comment (klappradla) [reply app/models/application.rb]: I don't remember the exact details of the selection process. But I'd say they indeed need to show up in the table as I don't see another way for reviewers to have an overview on which teams were selected.

The actual "reviews" are based on each reviewer's `todos` which is a second list view. The table shows all the teams with their points, review status etc. and is then the place where teams will be selected.

Comment (ramonh) [reply app/models/application.rb]: That makes sense! I'll make this change. Thanks for the feedback

**Comment by ramonh** [comment]
Should be good to go. Thanks for the feedback! I'll go ahead and merge :)

**Pull request merged**